# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: layout/reftests/bugs/reftest.list
# Commit: 867abe6c80a3
# Full Hash: 867abe6c80a36d66fec92f276c3a7d7c315637bf
# Author: Mats Palmgren <mats@mozilla.com>
# Date: 2021-03-01 09:36:12
# Description:
#   Bug 1694459 - Deal with 'display:contents' style changes on <fieldset> descendants better where it affects which <legend> is the rendered legend.  r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D106566
# ==============================================================================

diff -r 77247863a2ea -r 867abe6c80a3 layout/reftests/bugs/reftest.list
--- a/layout/reftests/bugs/reftest.list	Mon Mar 01 01:16:36 2021 +0000
+++ b/layout/reftests/bugs/reftest.list	Mon Mar 01 03:42:41 2021 +0000
@@ -873,7 +873,6 @@
 == 403656-4.html 403656-4-ref.html
 == 403656-5.html 403656-5-ref.html
 #== 403657-1.html 403657-1-ref.html  # Fails depending on the fonts...
-== 403733-1.html 403733-1-ref.html
 == 404030-1.html 404030-1-ref.html
 != 404030-1-notref.html 404030-1.html
 != 404030-1-notref2.html 404030-1.html